fission-web-server/library/Fission/Web/Server/Types.hs

```diff
@@ -632,7 +636,8 @@ instance MonadEmail Server where
         , params = Email.TemplateOptions verifyUrl name
         }
 
-    liftIO $ runClientM (Email.sendEmail apiKey emailData) env
+    mapLeft Email.CouldNotSend <$>
+      liftIO (runClientM (Email.sendEmail apiKey emailData) env)
```

Comment:
Also, no need to change, but I used to write a lot of this kind of code. Coming back many months later means having to reconstruct what's actually happening.

```haskell
liftIO (runClientM (Email.sendEmail apiKey emailData) env) >>= \case
  Left clientErr -> return $ Left Email.CouldNotSend clientErr
  Right _ -> return ok
```

This is often just clearer, if more LOC.